plugins/missions.js

```diff
@@ -238,7 +363,7 @@ window.plugin.missions = {
 				}
 				return;
 			}
-			callback(missions);
+			callback(missions, "Missions in View");
```

Comment:
Use single quotes https://github.com/IITC-CE/ingress-intel-total-conversion/wiki/Code-Style

Reply:
I made another commit to fix all the style guide issues